docs/index.html

```diff
@@ -179,8 +204,8 @@
     </article>
     <article><h1>Flexibility</h1>
         <p>Nakadi instances are stateless. They can be run on AWS with auto-scaling.</p>
-        <p>Using Timelines it is easy to move the traffic to another cluster without moving the data and any service
-            degradation.</p>
+        <p>Using Timelines it, is easy to move the traffic to another cluster without moving the data,
```

Comment:
Don't mess with Doctor Lionel Montrieux! The comma is needed. https://github.com/zalando/nakadi/pull/714#discussion_r128979152
 
But the place is wrong, my bad.